src/seckrit.py

```diff
@@ -10,11 +10,25 @@
 
 
 def main(args):
+
     # Load the YAML file that tells us which secrets we want to load (and what to do with them).
-    manifest = load_manifest(args.manifest_file)
+    if args.manifest:
+        manifest = load_manifest_from_file(args.manifest)
+    elif "SECKRIT_MANIFEST" in os.environ:
+        manifest = load_manifest_from_string()
+    else:
+        print("ERROR: Should pass --manifest argument or define SECKRIT_MANIFEST environment variable")
```

Comment:
Minor wording tweak:
```suggestion
        print("ERROR: You must either pass the --manifest argument or define the SECKRIT_MANIFEST environment variable")
```